alignak/daemons/schedulerdaemon.py

```diff
@@ -401,6 +538,10 @@ def main(self):
             # Look if we are enabled or not. If ok, start the daemon mode
             self.look_for_early_exit()
 
+            # todo:
+            # This function returns False if some problem is detected during initialization
+            # (eg. communication port not free)
+            # Perharps we should stop the initialization process and exit?
```

Comment:
Yes, but it's a TODO comment ... let's keep some job for later! It is currently too heavy to do this properly